src/ResourceManager/Reservations/Commands.Reservations.Test/ScenarioTests/ReservationOrderTests.ps1

```diff
@@ -0,0 +1,47 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Get a single reservation
+#>
+function Test-GetReservationOrder
+{
+	$reservationOrderId = "55793bc2-e5c2-4a98-9d5c-0a0bce6cf998"
+    $reservation = Get-AzureRmReservationOrder -ReservationOrderId $reservationOrderId
```

Comment:
what setup needs to happen in order to make this test pass?  Contributors will need to be able to run this test live, so any necessary setup must happen as part of the test

Reply:
The question has been answered previously, but currently we do not provide "create" reservation so cannot generate new reservation id from Powershell. That is why I am manually hardcoding this value, because we want to test other calls given specific reservation id.